bedrock/mozorg/templates/mozorg/rise25/landing.html

```diff
@@ -39,9 +39,16 @@ <h2 class="r25-fancy-title">AI for the people</h2>
       dedicated to supporting responsible AI initiatives that benefit humanity. Know
       an innovator making a difference with AI?</p>
     <div class="r25-hero-cta">
+    {% if switch('rise25-nominations-open') %}
       <strong class="r25-hero-notice">Call for nominees extended to<br> <time datetime="2024-04-12">April 12, 2024</time></strong>
 
       <a href="#nominate" class="r25-c-button t-fancy">Nominate <span>someone</span></a>
+    {% else %}
+      <p class="r25-hero-closed">
+        <strong>Nominations are now closed.</strong>
+        Follow us @Mozilla and stay tuned for the announcement of the Rise25 award winners.
```

Comment:
**Nit:** I don't think this is clear for users on which socials to follow? Maybe we can link Twitter?
```suggestion
        Follow us <a href="twitter.com/mozilla/">@Mozilla</a> and stay tuned for the announcement of the Rise25 award winners.
```

Reply:
I had that same thought but Rob says no link, since that's our handle on various platforms. So hopefully people can figure it out:

![image](https://github.com/mozilla/bedrock/assets/205591/1fa933c4-e85f-4a40-9228-16c97257fd4e)